MIDI_CLUE_BLE_Glove/midi_clue_ble_glove.py

```diff
@@ -0,0 +1,316 @@
+"""
+CLUE BLE MIDI
+Sends MIDI CC values based on accelerometer x & y and proximity sensor
+Touch #0 switches Bank/Preset patches
+Touch #1 picks among the three CC lines w A&B buttons adjusting CC numbers
+Touch #2 starts/stops sending CC messages (still allows Program Change)
+"""
+import time
+from adafruit_clue import clue
+import adafruit_ble
+from adafruit_ble.advertising.standard import ProvideServicesAdvertisement
+import adafruit_ble_midi
+import adafruit_midi
+from adafruit_midi.control_change import ControlChange
+from adafruit_midi.program_change import ProgramChange
+
+# from adafruit_midi.note_on import NoteOn
+# from adafruit_midi.pitch_bend import PitchBend
+import simpleio
+import displayio
+import terminalio
+from adafruit_display_text import label
+from adafruit_display_shapes.rect import Rect
+
+# --- Pick your midi out channel here ---
+midi_channel = 1
+# --- Pick your MIDI CC numbers here ---
+cc_x_num = 71
+cc_y_num = 72
+cc_prox_num = 73
+# --- Pick Bank & Preset pairs here ---
+touch_patch = [  # first number is the Bank, second is the Preset
+    (4, 16),  # minimoog: Leads > Original MINI
+    (5, 8),  # Pads > Intergalactic Pass
+    (0, 13),  # Bass > Kraft Bass
+    (6, 9),  # Percussion > Space Hat
+]
+
+patch_count = len(touch_patch)
+patch_index = (
+    patch_count - 1
+)  # start on the last one so first time it is pressed it goes to first
+
+cc_x = 0
+cc_y = 0
+cc_prox = 0
+
+# Use default HID descriptor
+midi_service = adafruit_ble_midi.MIDIService()
+advertisement = ProvideServicesAdvertisement(midi_service)
+
+ble = adafruit_ble.BLERadio()
+if ble.connected:
+    for c in ble.connections:
+        c.disconnect()
+
+midi = adafruit_midi.MIDI(midi_out=midi_service, out_channel=midi_channel - 1)
+
+print("advertising")
+ble.name = "CLUE BLE MIDI"
+ble.start_advertising(advertisement)
+
+clue.display.brightness = 1.0
+clue.pixel.brightness = 0.2
+screen = displayio.Group(max_size=17)
+
+ORANGE = 0xCE6136
+GRAY = 0x080808
+BLACK = 0x121212
+BLUE = 0x668190
+SILVER = 0xAAAAAA
+BROWN = 0x805D40
+
+# --- Setup screen ---
+# BG
+color_bitmap = displayio.Bitmap(240, 240, 1)
```

Comment:
instead of setting up a screen, can you use the generic clue text display?

Reply:
@ladyada I'm not sure if I can do different text sizes and horizontal lines and such with the `clue.simple_text_display` can I?

Reply:
![IMG_3054](https://user-images.githubusercontent.com/2192828/86500982-e1fdf280-bd49-11ea-856a-f02ce0d901aa.JPG)

Reply:
I checked in with @kattni and she thought it might not be possible to do some of the things I'm doing here using `clue.simple_text_display`

Reply:
hmm yeah thats kinda fancy ok!